src/index.js

```diff
@@ -34,19 +39,22 @@ module.exports = {
       return [
         ...fullCorvidTypes.BASE,
         ...fullCorvidTypes.BACKEND,
+        ...getJswTypeDeclarations(),
         ...getNpmDependenciesTypesDeclarations(dependencies)
       ];
     },
     public: ({ dependencies }) => {
       return [
         ...fullCorvidTypes.BASE,
         ...fullCorvidTypes.PUBLIC,
+        ...getJswTypeDeclarations(),
         ...getNpmDependenciesTypesDeclarations(dependencies)
       ];
     }
   },
   getWixModulesList: () => wixModulesNames,
   getWidgetTypeDeclarations: getWidgetDeclaration,
   getPageElementsTypeDeclarations: getPageElementsRawDeclarations,
-  getAmbientModuleDeclaration
+  getAmbientModuleDeclaration,
+  getJswTypeDeclaration: getJswRawDeclaration
```

Comment:
why do we need to expose this function ? it should be internal

Reply:
add comment/function to explain that this is used by the cli.
maybe don't add the jsw until it is used in the cli.
(we need to organize the cli's api)

Reply:
Done

Reply:
Added a comment but we still need these exports for the tests.
We run every test for the editor (in which we get the typings via the page/public/backend method) and another as if we are using the CLI (in which we use the exported functions to generate the typings and see that they work)